setup.py

```diff
@@ -1,8 +1,44 @@
 import os
+import re
+import sys
 import shutil
 from setuptools import setup, find_packages
 from setuptools.command.build_py import build_py
 from glob import glob
+ 
+# Determine if the user has specified which paths to report coverage for
+is_include_arg = map(lambda x: bool(re.search('^--include=', x)), 
+                     sys.argv)
+
+if True in is_include_arg:
+    include_arg = sys.argv[is_include_arg.index(True)]
+    include_arg = sys.argv[is_include_arg.index(True)]
+    del sys.argv[is_include_arg.index(True)]
+else:
+    include_arg = None
+
+
+class TestRepo(build_py):
+    '''Build command for running tests in repo'''
+    def run(self):
+        if include_arg:
+            coverage_command = 'coverage report -m %s' % include_arg
+        else:
+            coverage_command = 'coverage report -m --omit=setup.py,*/__init__.py'
```

Comment:
I think so, as we test neither our tests nor `./eggs` files and the "results" for these files clutter the ultimate report. Let me know if you disagree.